lib/internal/test_runner/harness.js

```diff
@@ -170,8 +170,20 @@ function runInParentContext(Factory) {
   return cb;
 }
 
+function afterEach(fn, options) {
+  const parent = testResources.get(executionAsyncId()) || setup(root);
+  parent.createBeforeEachHook(fn, options);
```

Comment:
Hi there, thanks for the contribution.
Q: Should this be `createAfterEachHook` instead?